EventsExpress/ClientApp/src/components/helpers/eventHelper.js

```diff
@@ -35,6 +35,10 @@ const eventHelper = (function () {
                 dateTo: undefined,
                 categories: [],
                 status: 'active',
+                selectedPos: null,
```

Comment:
Why do you need x, y and selectedPos in a filter? it should be either that or that.